ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/PackageUploadLedgerEntry.scala

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.platform.sandbox.stores.ledger
+
+import java.time.Instant
+
+import com.daml.ledger.participant.state.v1.ParticipantId
+
+//TODO BH consider removing duplication of this class
+sealed abstract class PackageUploadLedgerEntry() extends Product with Serializable {
+  val submissionId: String
+  val participantId: ParticipantId
+  val recordTime: Instant
+  def value: String
```

Comment:
`value` is not a particularly descriptive name. Please follow how it was done for `ConfigurationEntry`.